ha_cluster_exporter.go

```diff
@@ -94,6 +94,21 @@ func main() {
 		prometheus.MustRegister(drbdCollector)
 	}
 
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`<html>
+			<head>
+				<title>HACluster Exporter</title>
+			</head>
+			<body>
+				<h1>HACluster Exporter</h1>
+				<p><a href="/metrics">Metrics</a></p>
+				<br />
+				<h2>More information:</h2>
+				<p><a href="https://github.com/ClusterLabs/ha_cluster_exporter">github.com/ClusterLabs/ha_cluster_exporter</a></p>
+			</body>
+			</html>`))
+	})
```

Comment:
could you extract the handler function into a dedicated one? I'd like to keep the main as lean as possible.